rca/editorial/models.py

```diff
@@ -123,14 +158,16 @@ class EditorialPage(ContactFieldsMixin, BasePage):
         FieldPanel("published_at"),
         MultiFieldPanel(
             [
+                InlinePanel("related_schools", label="Related Schools"),
```

Comment:
switched to use the already exposed related school page on https://github.com/torchbox/rca-wagtail-2019/blob/master/rca/schools/models.py#L51